lxd/images.go

```diff
@@ -2572,8 +2572,8 @@ func imageDelete(d *Daemon, r *http.Request) response.Response {
 		return nil
 	}
 
-	resources := map[string][]string{}
-	resources["images"] = []string{imgInfo.Fingerprint}
+	resources := map[string][]url.URL{}
+	resources["images"] = []url.URL{api.NewURL().Path(imgInfo.Fingerprint).URL}
```

Comment:
We shouldn't be relying on the map key to be generating the URL anymore for this PR (that effectively becomes the resource category), as it wasn't always correct or was restrictive in terms of generating the actual URL to the resource. 

Instead I would expect to see the URLs being added to the slice to contain the full path to the resource.